ddl/schema.go

```diff
@@ -135,14 +137,14 @@ func getIDs(tables []*model.TableInfo) []int64 {
 }
 
 func (d *ddl) delReorgSchema(t *meta.Meta, job *model.Job) error {
+	var startKey kv.Key
 	var tableIDs []int64
-	if err := job.DecodeArgs(&tableIDs); err != nil {
-		// arg error, cancel this job.
+	if err := job.DecodeArgs(&tableIDs, &startKey); err != nil {
```

Comment:
If the origin job's arguments are without `startKey`, the decoding out  `startKey` value is nil. And if the value of `startKey` is nil,  the `startKey` will be set to `tablecodec.EncodeTablePrefix(id)`.